app/__init__.py

```diff
@@ -25,10 +25,11 @@ def null_to_dash(value):
     return value
 
 
-def clean_tags(text):
+def clean_tags(text, highlight_tag):
```

Comment:
docstring definitely needs updating, now mark has been generalised to highlight_tag and also add info about how it then replaces with mark.

function_name should probably be updated too
maybe
`clean_tags_and_replace_highlight_tag_with_mark`